string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,12 @@
+/* exported capitalize */
+function capitalize(word) {
+  var capitalStr = '';
+  for (var i = 0; i < word.length; i++) {
+    if (i === 0) {
+      capitalStr += word[i].toUpperCase();
+    } else {
```

Comment:
Instead of doing this in the loop, why not just capitalize the first letter before the loop, then start your loop from `1`?